zahra-portfolio.html

```diff
@@ -260,14 +260,16 @@ <h2 class="contact-title">Contact me</h2>
                 <li><input type="email" name="email" id="email" placeholder="your email here" required></li>    
                 <li><textarea id="msg" name="msg" placeholder="add your comment here" required minlength="1" maxlength="500"></textarea></li>
             </ul>
-            <p class="warning" id="warning"></p>  
-            <button class="button" id="submit-btn" type="submit">Get In Touch</button>
+            <p class="warning" id="warning"></p> 
+            <button class="form-button" id="submit-btn" type="submit">Get In Touch</button>
+            <button class="form-button" id="reset-btn" type="reset">Reset Data</button>
```

Comment:
- [ ] [OPTIONAL] Nice implementation, but you can delete this button "Reset data", it is not mandatory and does not cover Figma design.